packages/atomic/src/components/search/atomic-ipx-layout/ipx-layout.ts

```diff
@@ -0,0 +1,61 @@
+import {sectionSelector} from '../../common/atomic-layout-section/sections';
+
+const pagerSelector = 'atomic-pager';
+
+export function buildIPXLayout(element: HTMLElement) {
```

Comment:
Since you're not using the dynamic breakpoint (like the search layout uses) you won't need this .ts file. You can put all the styling in an atomic-ipx-layout.pcss. It's much more readable and maintainable

Reply:
Oh right! Will do thanks